docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Set up Automatic Agent Updates
+description: Describes how to set up automatic agent updates.
+---
+
+Teleport supports automatic agent updates for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers, as well as Kubernetes
+clusters. 
+
+This guide explains how to enable automatic updates for Teleport agents.
+
+## How it works
+
+When automatic updates is enabled, a Teleport updater is installed alongside
+each Teleport agent. The updater communicates with the Teleport Proxy Service to
+determine when an update is available. When an update is available, the updater
+will update the Teleport agent during the next maintenance window. However, if a
+critical update is availble, the Teleport agent will be updated outside the
+regular maintenance window.
+
+## Prerequisites
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components in your cluster.
+
+- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+Systemd agents enrolled into automatic updates can only install versions
+present in their package repositories. As Teleport 14 won't be published to
+`stable/v13`, those agents will require manual intervention to be upgraded to
+the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
+
+Alternatively, you can use the `stable/rolling` channel, which contains
+Teleport v13.3.2 forward, including future major releases.
+
+## Step 1/3. Find agents to enroll in automatic updates
+
+Use the `tctl inventory ls` command to list connected agents along with their current
+version. Use the `--upgrader=none` flag to list agents that are not enrolled in automatic
+updates.
+
+
+```code
+$ tctl inventory ls --upgrader=none
+Server ID                            Hostname      Services Version Upgrader
+------------------------------------ ------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 ip-10-1-6-130 Node     v14.4.5 none
+...
+```
+
+Note that the `inventory ls` command will also list `teleport-auth` and
+`teleport-proxy` services. These services are managed by the Teleport team, and
+updates will be performed automatically.
```

Comment:
The auth and proxy sevrices are only managed by us for cloud customers, self-hosted AU users are also responsible for updating their control plane services.

Reply:
I had intended for this to be a guide for cloud-hosted clusters, but now realize that nothing in the introduction/prerequisites mentions this. I've added a prerequisite for a cloud-hosted cluster and a link to the reference for self-hosted users.